tests/test_genomes.py

```diff
@@ -0,0 +1,87 @@
+"""Integration tests for Genomes Pipeline
+
+These tests verify that reference genome data can be extracted from upstream
+sources (e.g. NCBI, Ensembl) transformed, as expected by code that loads
+transformed data to Google Cloud Storage.
+
+Test doubles are used for test speed and isolation.
+
+PREREQUISITES
+See https://github.com/broadinstitute/scp-ingest-pipeline#prerequisites
+
+Also, `cd tests` before running the examples below.
+
+EXAMPLES
+
+# Run all tests
+pytest
+
+# Run all tests and see print() output
+pytest -s
+
+# Run only tests in test_genomes.py
+pytest test_genomes.py
+
+# Run all tests, show code coverage metrics
+pytest --cov=../ingest/
+
```

Comment:
I think it's overkill to be repeating pytest usage documentation here.    Maybe just add a link to the pytest docs from the main readme (line 76ish)?

Reply:
Yeah, I can see how repeating these examples in each integration test module is a bit much.  At the same time, I think only linking to pytest docs makes conveying tailored examples too inconvenient.

So in 8a486fe I've consolidated this handful of conveniences into the "Test" section of the README, added a pointer here, and removed the verbose examples from the tests themselves.